app/javascript/application.scss

```diff
@@ -37,3 +37,19 @@ $mdi-font-path: "~@mdi/font/fonts/";
 .white-space-pre-wrap {
   white-space: pre-wrap;
 }
+
+// Button background custum variant
+.btn-pink {
```

Comment:
Do we still need the pink and purple button classes if we're moving forward with the orange?

Reply:
They were added to experiment, so no longer needed. will remove. Thanks!